### PR TITLE
Content structure improvements - validation tasklist

### PR DESCRIPTION
**Pull request by benbaumann95**
### Description of change

- Please see commits for details

### Story Link

https://trello.com/c/t7Q2xjLL/996-content-and-structure-improvements-to-validation-tasklist

Screenshot depicting 4 invalid and 2 updated validation request items.
(FYI the 4 invalid items come from the red line boundary, a replacement document request and 2 missing document requests)

Screenshot 2022-06-22 at 10 38 49

### CODE REVIEW

File: app/views/planning_application/validation_tasks/_other_validation_request.html.erb [hunk 1]
```diff
@@ -12,10 +12,7 @@
     <% end %>
     <% @planning_application.other_change_validation_requests.non_fee_item.not_cancelled.each do |other_change_validation_request| %>
       <li class="app-task-list__item">
-        <span class="app-task-list__task-name">
-          <%= link_to "View other validation request ##{other_change_validation_request.sequence}",
-            planning_application_other_change_validation_request_path(@planning_application, other_change_validation_request), class: "govuk-link" %>
-        </span>
+        <%= @planning_application.other_change_task_list(other_change_validation_request) %>
```
Comment (allbecauseyoutoldmeso) [inline app/views/planning_application/validation_tasks/_other_validation_request.html.erb:15]: 🎉

File: app/presenters/concerns/validation_tasks/review_presenter.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module ValidationTasks
+  extend ActiveSupport::Concern
+
+  class ReviewPresenter < PlanningApplicationPresenter
+    def task_list_row
+      html = tag.span class: "app-task-list__task-name" do
+        concat link_to "Send validation decision",
+                       validation_decision_planning_application_path(@planning_application), class: "govuk-link"
```
Comment (allbecauseyoutoldmeso) [inline app/presenters/concerns/validation_tasks/review_presenter.rb:10]: Again I think no need for the `@`

File: app/views/planning_applications/validation_decision.html.erb [hunk 3]
```diff
@@ -38,17 +38,14 @@
           <%= "The application is marked as invalid. The applicant was notified on #{@planning_application.invalidated_at}" %>
         </strong>
       </p>
-      <p class="govuk-body">
-        <%= link_to "Check documents", planning_application_documents_path(@planning_application), class: "govuk-link" %>
-      </p>
```
Comment (allbecauseyoutoldmeso) [inline app/views/planning_applications/validation_decision.html.erb:43]: Is removing this intentional? I couldn't see it in the ACs but there's a lot going on so fine if it is 😅

**Review by allbecauseyoutoldmeso — APPROVED** [review]
Loving the use of presenters!

And thanks for splitting this up into logical PRs - it made it much easier to review!

File: app/views/planning_applications/validation_decision.html.erb (hunk 3 above)
Comment (benbaumann95) [reply app/views/planning_applications/validation_decision.html.erb:43]: Going off the design prototypes and I think I cleared it with Tom.
I'll double check for sure, thanks for asking!

**Pull request merged**